.github/CODEOWNERS

```diff
@@ -0,0 +1,135 @@
+
+
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @rrsettgast
+
+/.github/ @rrsettgast
+
+/inputFiles/ @CusiniM @cssherman @jhuang2601
+
+/src/coreComponents/LvArray                           @rrsettgast @wrtobin
+/src/coreComponents/codingUtilities                   @rrsettgast
+/src/coreComponents/common                            @rrsettgast
+/src/coreComponents/constitutive                      @rrsettgast
+/src/coreComponents/dataRepository                    @rrsettgast @wrtobin
+/src/coreComponents/denseLinearAlgebra                @rrsettgast
+/src/coreComponents/discretizationMethods             @rrsettgast
+/src/coreComponents/events                            @rrsettgast
+/src/coreComponents/fieldSpecification                @rrsettgast
+/src/coreComponents/fileIO                            @rrsettgast @wrtobin
+/src/coreComponents/finiteElement                     @rrsettgast
+/src/coreComponents/finiteVolume                      @rrsettgast
+/src/coreComponents/functions                         @rrsettgast
+/src/coreComponents/linearAlgebra                     @rrsettgast
+/src/coreComponents/mainInterface                     @rrsettgast
+/src/coreComponents/math                              @rrsettgast
+/src/coreComponents/mesh                              @rrsettgast @wrtobin
+/src/coreComponents/physicsSolvers                    @rrsettgast
+/src/coreComponents/physicsSolvers/contact            @CusiniM
+/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM
```

Comment:
@paveltomin @castelletto1 ?

Reply:
just add it

Reply:
> @paveltomin @castelletto1 ?

looks good
i would refine constitutive and multiphycs solvers a bit more if possible but that can be done later